label_studio/tasks/models.py

```diff
@@ -1291,9 +1291,23 @@ def remove_predictions_from_project(sender, instance, **kwargs):
     """Remove predictions counters"""
     instance.task.total_predictions = instance.task.predictions.all().count() - 1
     instance.task.save(update_fields=['total_predictions'])
+
+    # if there is PredictionMeta object associated with the Prediction object, delete it
+    if hasattr(instance, 'meta'):
```

Comment:
would it be more robust to filter on PredictionMeta table for this id instead of checking the 'meta' attr, as is done in https://github.com/HumanSignal/label-studio/blob/develop/label_studio/ml_models/models.py#L179?

Reply:
can you elaborate please, why is it more robust?